services/core-web/src/reducers/staticContentReducer.js

```diff
@@ -20,7 +20,7 @@ const initialState = {
   incidentStatusCodeOptions: [],
   incidentCategoryCodeOptions: [],
   varianceStatusOptions: [],
-  varianceDocumentCategoryOptions: [],
+  documentCategoryOptions: [],
```

Comment:
are these document categories generic, why not keep the `variance` prefix?

Reply:
Not sure why I did that, I'll put that back in.

Reply:
Looks like that is done throughout the app in some Core files as well.

Reply:
I didn't mean to change it for that Core file so I put it back.